ruby-engine/build.sh

```diff
@@ -1,4 +1,45 @@
+#!/bin/bash
```

Comment:
You can mostly ignore this, it's a utility script so you don't have to manually copy the binaries into the Ruby folder for local dev

Reply:
I would set it to fail if something goes wrong
```suggestion
#!/bin/bash
set -e
```

Reply:
Thanks! Good suggestion!